ocrd/ocrd/decorators.py

```diff
@@ -46,9 +56,21 @@ def ocrd_cli_wrap_processor(processorClass, ocrd_tool=None, mets=None, working_d
             raise Exception(msg)
         resolver = Resolver()
         workspace = resolver.workspace_from_url(mets, working_dir)
-        # TODO once we implement 'overwrite' CLI option and mechanism, disable the
-        # `output_file_grp_ check by setting to False-y value if 'overwrite' is set
-        report = WorkspaceValidator.check_file_grp(workspace, kwargs['input_file_grp'], kwargs['output_file_grp'])
+        page_id = kwargs.get('page_id')
+        if overwrite:
+            if 'output_file_grp' not in kwargs or not kwargs['output_file_grp']:
+                raise Exception("--overwrite requires --output-file-grp")
+            LOG.debug("Removing files because of --overwrite")
```

Comment:
I am currently testing on real workflows. One thing that sticks out is that this log message is only _debug_ level, while the subsequent messages (up to 3 per file ID) are all _info_: 
- in `ocrd_models.ocrd_mets`: `remove_one_file` ...
- in `ocrd_models.ocrd_mets`: `Delete fptr element` ...
- in `ocrd.workspace`: `rm ` ...

So (to give these other messages at least some context) IMHO either this should be upgraded to _info_ as well, or the others downgraded to _debug_.

Reply:
```suggestion
            LOG.info("Removing files because of --overwrite")
```